packages/builder/dist/electron/build.sh

```diff
@@ -22,9 +22,13 @@ set -o pipefail
 #
 # @param $1 staging directory
 # @param $2 platform (optional) defaulting to all platforms (you may also set this via PLATFORM env)
+# @param $3 client directory (optional) defaulting to default
 #
 STAGING="${1-`pwd`}"
 PLATFORM=${2-${PLATFORM-all}}
+export CLIENT_NAME="${3-Kui}"
```

Comment:
shouldn't the default value by `$PRODUCT_NAME`, not hard-wired?

Reply:
Updated, thanks!